### DISCUSSION TITLE
All possible results for "textDescription" field from a '/stations/<station>/observations" API call?

### DISCUSSION THREAD

**Discussion by EdwardDeaver** [weather-gov/api#761] [Q&A]
Hello, 

What are the possible results of the "textDescription" field in a  '/stations/<station>/observations" API call?

Example: "https://api.weather.gov/stations/KSYR/observations" 

`"station": "https://api.weather.gov/stations/KSYR",
                "timestamp": "2024-08-23T20:54:00+00:00",
                "rawMessage": "KSYR 232054Z 29007KT 10SM SCT060 SCT070 BKN270 26/12 A3025 RMK AO2 SLP238 T02560122 56009",
                "textDescription": "Mostly Cloudy",
                "icon": "https://api.weather.gov/icons/land/day/bkn?size=medium",`

What are the other statuses that textDescription can have?

**Comment by dwhitemv25** [comment]
This is a FAQ, answered in #386, #547, #745, et. al.
